docs/syntax/origamijson.md

```diff
@@ -96,37 +96,29 @@ All origami components, whether modules or web services, should be discoverable
 	<td>string</td>
 	<td>(optional) For web services only, the URL on which the service is provided.  Required for web services.</td>
 </tr><tr>
+	<td><code>&nbsp;&nbsp;demosDefaults:&nbsp;{</code></td>
+	<td>object</td>
+	<td>(optional) Default options to be applied to all demos. Please check out the different options in the [modules component spec](http://origami.ft.com/docs/component-spec/modules/#demo-config)</td>
```

Comment:
I gave it some thought. My initial idea was to put it in both, and then just here, and I actually started doing that, but I thought it was too messy. It makes the table bigger and harder to comprehend, and I don't think it makes the idea of default options can also be applied to individual demos come across as nicely.